requirements.txt

```diff
@@ -2,4 +2,5 @@ matplotlib==3.7.3
 numpy==1.24.4
 pandas==1.5.3
 scikit_learn==1.3.2
-tables==3.8.0
\ No newline at end of file
+tables==3.8.0
+python-dotenv~=1.0.1
```

Comment:
What is this requirement?

Reply:
not sure what the tables was for. The dotenv is used to use the .env files to load the hardcoded filepaths